### PR TITLE
fix: remove affix props from textfields own props

### PR DESCRIPTION
**Pull request by AnnaRybkina**
Background: https://sch-chat.slack.com/archives/C04P0GYTHPV/p1688627911916319

- [x] Moved out affix props from Input and Textfield props
- [x] Add missing affix props to the Elements.md
- [x] Added a disclaimer about the `type` of the Affix

### CODE REVIEW

File: docs/components/textfield/react.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ Then you include it as a child of TextField component and pass the appropiate pr
 </TextField>
 ```
 
-## Affix props
+You cannot set the `type` of the Affix. `type` of the button renderered as `Affix` will be `submit` if you pass `search` as a prop, `reset` if you pass `clear` and `undefined` in other cases.
```
Comment (BalbinaK) [inline docs/components/textfield/react.md]: *rendered 🙃

Comment (AnnaRybkina) [reply docs/components/textfield/react.md]: 👁️

**Review by BalbinaK — APPROVED** [review]
💪

File: docs/components/textfield/react.md (hunk 1 above)
Comment (AnnaRybkina) [reply docs/components/textfield/react.md]: Fixed https://github.com/warp-ds/tech-docs/pull/84/commits/91243c625e5b4a49ccef564db737f235b6fdfdfd

**Review by BalbinaK — APPROVED** [review]

**Pull request merged**